api/config/esgateway/config_request.proto

```diff
@@ -43,7 +43,14 @@ message ConfigRequest {
 				message Main {
 					google.protobuf.Int32Value worker_processes = 1;
 					google.protobuf.Int32Value max_fails = 2;
-					google.protobuf.StringValue resolvers = 3;
+					reserved 3;
```

Comment:
i think we're saying this breaking change is ok because we haven't released this "resolvers" field to the world?

Reply:
Yes indeed. I added as the build was failing since there is a change in the filed format.